tests/Main.hs

```diff
@@ -95,7 +95,7 @@ propTest prop = do
     GaveUp { output = o } -> (False, o)
     Failure { output = o } -> (False, o)
     NoExpectedFailure { output = o } -> (False, o)
-    InsufficientCoverage { output = o } -> (False, o)
+--  InsufficientCoverage { output = o } -> (False, o)
```

Comment:
Is commenting out the right thing, or should this code be removed?

Reply:
Probably removing altogether.